supervision/detection/core.py

```diff
@@ -483,11 +484,40 @@ class names. If provided, the resulting Detections object will contain
             Class names values can be accessed using `detections["class_name"]`.
```

Comment:
We need to update the `from_transformers` docstring. 

>  Creates a Detections instance from object detection or segmentation [Transformer](https://github.com/huggingface/transformers) inference result.

I'd mention we support object detection as well as panoptic, semantic and instance segmentation results.

> transformers_results (dict): The output of Transformers model inference. A dictionary containing the `scores`, `labels`, `boxes` and `masks` keys.

Now it can also be a `Tensor`.

Reply:
Hi @SkalskiP , One quick query, ideally it should now be `transformers_results (Union[dict, torch.Tensor])` but to support this we need to import the torch package, should we add the import or leave it in docstrings for now?